src/openms/include/OpenMS/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderAlgorithmMetaboIdent.h

```diff
@@ -88,7 +88,7 @@ class OPENMS_DLLAPI FeatureFinderAlgorithmMetaboIdent :
   FeatureFinderAlgorithmMetaboIdent();
 
   /// @brief perform targeted feature extraction of compounds from @param metaboIdentTable and stores them in @param feature
```

Comment:
```suggestion
  /// @brief perform targeted feature extraction of compounds from @param metaboIdentTable and stores them in @param feature.
  /// If @p spectra_file is provided it will be used as a fall-back to setPrimaryMSRunPath
  /// in the feature map in case a proper primaryMSRunPath is not annotated in the MSExperiment.
```